packages/core/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # @web3inbox/core
 
+## 1.0.0
+
+### Major Changes
+
+- Stable release
```

Comment:
Maybe have a longer description for the first major release?

Reply:
I feel like it's way too much for a change log since essentially *everything* changed. Documenting everything that changed would be like pasting docs in here

Reply:
What would you put?

Reply:
@devceline I'll be sharing shortly

Reply:
Let's skip this for now